docs/content/concepts/scenarios/kind-scenario/README.md

```diff
@@ -0,0 +1,613 @@
+# List KCP Workspace objects (Kind cluster scenario)
+
+Table of content:
+
+- [List KCP Workspace objects (Kind cluster scenario)](#list-kcp-workspace-objects-kind-cluster-scenario)
+  - [1. Start KCP](#1-start-kcp)
+  - [2. Check objects in the new `kind` KCP Workspace at the center (#1)](#2-check-objects-in-the-new-kind-kcp-workspace-at-the-center-1)
+  - [3. Create a syncer at the center](#3-create-a-syncer-at-the-center)
+  - [4. Check objects that were created in the `kind` KCP Workspace after running `kcp workload sync` at the center (#2)](#4-check-objects-that-were-created-in-the-kind-kcp-workspace-after-running-kcp-workload-sync-at-the-center-2)
+  - [5. Apply the syncer on the pcluster at the edge (Kind cluster)](#5-apply-the-syncer-on-the-pcluster-at-the-edge-kind-cluster)
+  - [6. Check objects that were created in the `kind` KCP Workspace at the center after running the syncer in the pcluster at the edge (#3)](#6-check-objects-that-were-created-in-the-kind-kcp-workspace-at-the-center-after-running-the-syncer-in-the-pcluster-at-the-edge-3)
+  - [7. Bind to `compute` at the center](#7-bind-to-compute-at-the-center)
+  - [8. Check objects that were created in the `kind` KCP Workspace after binding to `compute` at the center (#4)](#8-check-objects-that-were-created-in-the-kind-kcp-workspace-after-binding-to-compute-at-the-center-4)
+
+## 1. Start KCP
+
+Given KCP 0.11:
+
+```bash
+$ k version --short
+Client Version: v1.25.3
+Kustomize Version: v4.5.7
+Server Version: v1.24.3+kcp-v0.11.0
+```
+
+Start KCP with an external IP address binding:
+
+```bash
+$ kcp start --bind-address $(ifconfig | grep -A 1 'enp0s8' | tail -1 | awk '{print $2}')
+```
+
+Create a new KCP Workspace called `kind`:
+
+```bash
+$ k ws create kind --enter
+Workspace "kind" (type root:organization) created. Waiting for it to be ready...
+Workspace "kind" (type root:organization) is ready to use.
+Current workspace is "root:kind" (type root:organization).
+
+$ k ws tree
+.
+└── root
+    ├── compute
+    ├── jn
+    └── kind
+```
+
+## 2. Check objects in the new `kind` KCP Workspace at the center (#1)
+
+Given:
+
+```bash
+$ echo $KUBECONFIG
+~/.kcp/admin.kubeconfig
+
+$ k ws .
+Current workspace is "root:kind".
+```
+
+List key objects:
+
+```bash
+$ k get apiexports
+No resources found
+
+$ k get apibindings
+NAME                       AGE
+apiresource.kcp.io-c8wp6   49m
+scheduling.kcp.io-7rr1j    49m

```

Comment:
This APIBinding is relevant, as is the APIExport that it references.  It would be helpful to show those too.

Reply:
@MikeSpreitzer good idea.
I have added all 5 objects to the PR commit.
Please take a look when you have time